src/index.html

```diff
@@ -1,13 +1,145 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page__body">
+    <div class="stars">
+      <ul class="stars__list stars--0">
+        <li class="stars__item">
+          <div class="stars__star stars__star--inactive"></div>
+        </li>
+
+        <li class="stars__item">
+          <div class="stars__star stars__star--inactive"></div>
+        </li>
```

Comment:
```suggestion
      <div class="stars stars--0">
        <div class="stars__star"></div>
       <div class="stars__star"></div>  
       ...
```
stars is not a list. use just div.
no need any classes for star
use stars--1, stars--2... modifiers should be responsible about amount of active stars. use nth-child for it.